### ISSUE TITLE
Stop forking tests

### ISSUE THREAD

**Issue by thpani** [informalsystems/apalache#1577]
We should remove `Test / fork := true` from `build.sbt`: https://github.com/informalsystems/apalache/pull/1553#discussion_r834524465

This will require some additional work:
- Without forking, sbt runs tests from the project root (instead of subproject directories). This breaks some tests with relative paths to `.tla` files in unit tests.
- [Forked tests run sequentially](https://www.scala-sbt.org/1.x/docs/Testing.html#Forking+tests). There are some unit tests that fail when run in parallel.

In summary:
- [x] Remove `Test / fork := true` #1705
- [x] Fix path-dependent unit tests #1705

**Comment by shonfeder** [comment]
We think maybe the pending work item dependent on SANY can be factored out into its own issue and this closed.

**Comment by thpani** [comment]
Closed by #1705 after factoring out #1761
